index.html

```diff
@@ -101,6 +101,78 @@ <h2>
         display modes. Otherwise, the [=manifest/display=] member is sufficient
         for most use cases.
       </p>
+
+      <section>
+        <h3>Concepts</h3>
+        <ul>
+          <li>
+            <dfn data-lt="controls of the window">Window controls</dfn>:
+            interface elements that the operating system uses consistently across
+            applications to enable the user to perform certain actions to control
+            the application. Common actions in the [=window controls=] include
+            minimize, maximize/restore, and close buttons.
+          </li>
+          <li>
+            <dfn>Additional Window Controls</dfn>: feature to replace
+            [=window controls=] by calling the minimize/maximize/close etc APIs
+            directly without the need to use browser's native window controls.
+          </li>
+          <li>
+            <dfn data-lt="IWA">Isolated Web App</dfn>: feature for installed
+            and signed web applications which are packaged as Web Bundles and
+            can be used offline instead of being fetched over HTTPS.
+            More: https://github.com/WICG/isolated-web-apps
+          </li>
+          <li>
+            <dfn data-lt="draggable region">Draggable regions</dfn>. To define
+            which regions or elements in the title bar are draggable, this
+            specification defines the CSS <code><dfn>app-region</dfn></code>
+            property.
+
+            For now, [=app-region=] has not been standardized, so until that
+            the prefixed <code><dfn>-webkit-app-region</dfn></code> shall be
+            used as the supported CSS property.
+          </li>
+        </ul>
+      </section>
+
+      <section>
+        <h3>
+          Display mode extensions
+        </h3>
+        <p>
+          Additionally to the normal
+          <a data-cite="appmanifest#dfn-display-modes-list">display modes</a>,
+          [=manifest/display_override=] also supports certain extensions to it.
+        </p>
+        
+        <dt>
+          <dfn data-export="" data-dfn-for="display mode">
+            borderless
+          </dfn>
+        </dt>
+        <dd>
+          The web application does not have any host-native title bar or
+          [=window controls=] visible and with the web contents extended to the
+          whole title bar area. The app can specify draggable regions in the web
+          contents to create a customized title bar. The user agent may change
+          the title bar state depending on various security considerations, like
+          an out-of-scope navigation.
+        </dd>
+
+        <dt>
+          <dfn data-export="" data-dfn-for="display mode">
+            window-controls-overlay
```

Comment:
My only worry here is that we already have a definition here: https://wicg.github.io/window-controls-overlay/#dfn-window-controls-overlay-0. And this document already correctly links there.

Without migrating the window controls overlay spec into this one, I think we should

- If possible, just link to common concepts in the window-controls-overlay spec. This might require you to export them from that spec.
- (if not, it's fine to duplicate them)

### Option 1: Keep the 'extensions' for display_override section

- Have this section link to window-controls-overlay, instead of defining it.
- Keep 'borderless' definition in here.
- Link to this section specifically in the "display_override member" section as extensions, instead of listing them again in the txt.

### Option 2: Change the 'extension' section to just 'borderless' display mode.

- Have this section just be describing the 'borderless' display mode (so it isn't designed to encompass all extensions) and remove the window-controls-overlay definition.

What do you think?

@mgiuca interested in your thoughts if you have cycles.

Reply:
IMO we should not have cyclic dependencies between these specs. I see these documents as "staging" drafts before they get merged upstream into the "parent" doc. The dependencies should be like this:

WCO -> Manifest Incubations -> Manifest

If Manifest Incubations has a cyclic dependency on WCO, then it implies the concepts in WCO are mature enough to be promoted into MI.

Ultimately, WCO should be entirely folded into MI. But to avoid biting off the entire burger at once, could we just migrate any concepts we need right now into MI? Thus I propose Option 3:

- Copy the common concepts from the WCO spec to MI. (Are we just talking about draggable regions, or something more?)
- Submit a pull request to WCO to remove those concepts and link to MI instead.

Reply:
SGTM! 

Here would be the corresponding changes to the WCO spec:
https://github.com/WICG/window-controls-overlay/compare/main...sonkkeli:window-controls-overlay:move-draggable-regions-to-manifest-incubations-spec?expand=1

I'll create a PR once this one is merged?

Reply:
That's good. Feel free to submit that other PR at the same time and point to the fact that it's being added to incubations. (I don't think it'll be controversial, given that it contains a paragraph that literally says it "might belong in other specifications".)